src/storage/table/struct_column_data.cpp

```diff
@@ -179,10 +211,11 @@ void StructColumnData::Skip(ColumnScanState &state, idx_t count) {
 
 	// skip inside the sub-columns
 	IterateFields(
-	    state, [&](ColumnData &field, optional_idx field_vector_index, ColumnScanState &field_state, bool should_scan) {
+	    state, [&](idx_t child_index, optional_idx field_vector_index, ColumnScanState &field_state, bool should_scan) {
```

Comment:
Is this change still necessary? I don't see the `child_index` used anywhere except for getting the field

Reply:
Yea true, it was initially used to check the type of the child ColumnData, can definitely revert that now